src/System.Reflection.MetadataLoadContext/Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\Directory.Build.props" />
+  <PropertyGroup>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <AssemblyKey>OPEN</AssemblyKey>
+    <IsNETCoreApp>false</IsNETCoreApp>
```

Comment:
You don't need this. This is false by default.

Reply:
Removed.